trento/adoc/trento-mcp-integration.adoc

```diff
@@ -0,0 +1,13 @@
+include::product-attributes.adoc[]
+
+
+[[sec-trento-mcp-integration]]
+:revdate: 2025-10-31
+
+== Integration with {tr_mcp}
```

Comment:
The feature is MCP integration. MCP Trento Server is the component that makes it possible. I suggest to rename the section title.
```suggestion
== MCP Integration
```